exercises/all-your-base/example.tt

```diff
@@ -4,8 +4,8 @@ require_relative 'all_your_base'
 # Test data version: <%= sha1 %>
 class AllYourBaseTest < Minitest::Test<% test_cases.each do |test_case| %>
   def <%= test_case.test_name %>
-    <%= test_case.skipped %>
-    <%= test_case.workload %>
+<%= test_case.skipped %>
+<%= test_case.workload %>
```

Comment:
By removing the leading whitespace in the template here, we make everything a lot simpler.